pages/dashboard.js

```diff
@@ -209,20 +207,19 @@ const Dashboard = () => {
                   {!userSessionCountLoading && (
                     <Text
                       aria-describedby='user-session-count'
-                      fontSize='2rem'
-                      fontWeight='bold'
+                      fontSize='2.5rem'
                       color='ocean'
                     >
                       {userSessionCount}
                     </Text>
                   )}
                   <Heading
                     as='h3'
-                    fontSize='1.15rem'
-                    fontWeight='500'
+                    fontSize='0.9rem'
+                    fontWeight='normal'
                     id='user-session-count'
                   >
-                    Installs Performed
+                    Installations performed
```

Comment:
you and melissa have conflicting changes - both pushed at 1am lol. i like this wording though